src/app.service.ts

```diff
@@ -12,10 +12,25 @@ export class AppService {
   constructor(private prisma: PrismaService) {}
 
   async getStats(): Promise<any> {
-    return this.prisma.trending.groupBy({
+    const stats = await this.prisma.trending.groupBy({
       by: ['createdAt', 'type'],
       _count: { id: true },
     });
+    if(stats.length > 0){
+      return stats
+    }else{
```

Comment:
I think the `}else{` is not needed because the `if` has a `return`

Reply:
Done

Reply:
Thanks